templates/user/settings/account.tmpl

```diff
@@ -152,6 +153,7 @@
 				</div>
 			</form>
 		</div>
+		{{end}}
```

Comment:
The `id="delete-account"` is still there below. Maybe that code block should be moved into this "segment"